model/src/main/kotlin/OrtResult.kt

```diff
@@ -58,22 +58,26 @@ data class OrtResult(
         val data: CustomData = emptyMap()
 ) {
     /**
-     * Return all projects and packages that are likely to belong to the vendor of the given [name]. Projects are
-     * converted to packages in the result. If no analyzer result is present an empty set is returned.
+     * Return all projects and packages that are likely to belong to the vendor of the given [name]. If [omitExcluded]
+     * is set to true, excluded projects / packages are omitted from the result. Projects are converted to packages in
+     * the result. If no analyzer result is present an empty set is returned.
      */
     @Suppress("UNUSED") // This is intended to be mostly used via scripting.
-    fun getVendorPackages(name: String): SortedSet<Package> {
+    fun getVendorPackages(name: String, omitExcluded: Boolean = false): SortedSet<Package> {
         val vendorPackages = sortedSetOf<Package>()
+        val excludes = repository.config.excludes.takeIf { omitExcluded }
 
         analyzer?.result?.apply {
             projects.filter {
-                it.id.isFromVendor(name)
+                val isExcluded = omitExcluded && excludes?.isProjectExcluded(it) == true
+                it.id.isFromVendor(name) && !isExcluded
             }.mapTo(vendorPackages) {
                 it.toPackage()
             }
 
-            packages.filter {
-                it.pkg.id.isFromVendor(name)
+            packages.filter { (pkg, _) ->
+                val isExcluded = omitExcluded && excludes?.isPackageExcluded(pkg.id, analyzer.result) == true
```

Comment:
`omitExcluded` is not required as `excludes?.isProjectExcluded(it)` is always `null` and therefore `!= true` if `omitExcluded` is false.